<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Background:rgb(99, 99, 99);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel  {
	color: white;
	font: &quot;MS Shell Dlg 2&quot;;
}

QLCDNumber {
	color: white;
	font: 25px &quot;MS Shell Dlg 2&quot;;
	border: none;
}</string>
   </property>
   <widget class="QLabel" name="signalStrength">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>50</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tempE_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>90</width>
      <height>115</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>120</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>40</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom-width: 3px;
border-right-width: 3px;
border-right-color: white;
border-bottom-color: white;
border-style: solid;
border-bottom-right-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Â°C</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignJustify</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>-2</number>
    </property>
   </widget>
   <widget class="QLabel" name="temp_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>151</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>50</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold;
border-top-width: 3px;
border-left-width: 3px;
border-left-color: white;
border-top-color: white;
border-style: solid;
border-top-left-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Temperatur</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="tempALCD">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>230</width>
      <height>115</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>120</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom-width: 3px;
border-left-width: 3px;
border-left-color: white;
border-bottom-color: white;
border-style: solid;
border-bottom-left-radius: 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="pressureLCD">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>550</y>
      <width>240</width>
      <height>115</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>110</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom-width: 3px;
border-left-width: 3px;
border-left-color: white;
border-bottom-color: white;
border-style: solid;
border-bottom-left-radius: 10px;</string>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="presE_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>550</y>
      <width>90</width>
      <height>115</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>120</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>40</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom-width: 3px;
border-right-width: 3px;
border-right-color: white;
border-bottom-color: white;
border-style: solid;
border-bottom-right-radius: 10px;</string>
    </property>
    <property name="text">
     <string>bar</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignJustify</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>-2</number>
    </property>
   </widget>
   <widget class="QLabel" name="pres_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>500</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>50</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold;
border-top-width: 3px;
border-left-width: 3px;
border-left-color: white;
border-top-color: white;
border-style: solid;
border-top-left-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Druck</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="placeholderTemp_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>170</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>50</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom-width: 3px;
border-left-width: 3px;
border-left-color: white;
border-bottom-color: white;
border-style: solid;
border-bottom-left-radius: 10px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignJustify</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="placeholderPres_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>50</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom-width: 3px;
border-left-width: 3px;
border-left-color: white;
border-bottom-color: white;
border-style: solid;
border-bottom-left-radius: 10px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignJustify</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="timeLeftS_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>730</y>
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>50</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>50</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold;
border-top-width: 3px;
border-left-width: 3px;
border-bottom-width: 3px;
border-left-color: white;
border-top-color: white;
border-bottom-color:white;
border-style: solid;
border-bottom-left-radius: 10px;
border-top-left-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Verbleibende Zeit </string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="timeLeftN_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>730</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>50</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold;
border-top-width: 3px;
border-bottom-width: 3px;
border-bottom-color: white;
border-top-color: white;
border-style: solid;
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="timeLeftE_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>730</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>50</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold;
border-top-width: 3px;
border-right-width: 3px;
border-bottom-width: 3px;
border-right-color: white;
border-bottom-color: white;
border-top-color: white;
border-style: solid;
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;</string>
    </property>
    <property name="text">
     <string>min</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="isMoving_Label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>330</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>330</width>
      <height>50</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>50</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold;
border-width: 3px;
border-color:white;
border-style: solid;
border-radius: 10px;

background-color: green;
</string>
    </property>
    <property name="text">
     <string>In Bewegung </string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
